pandas/core/categorical.py

```diff
@@ -118,6 +118,10 @@ def maybe_to_categorical(array):
 
 Assigning to `categories` is a inplace operation!
 
+Note
+----
+Categories getter returns all categories and not only those present in the data
```

Comment:
My point was to implement first bulet in [#12278 comment](https://github.com/pydata/pandas/issues/12278#issuecomment-182445522)
- [ ] add a note to the docstring of categories that this includes all categories and not only those present in the data

But I agree that it is a reduntant note